modules/python3/bindings/src/pyprocessors.cpp

```diff
@@ -89,13 +67,13 @@ class ProcessorFactoryObjectTrampoline : public ProcessorFactoryObject {
 
     virtual std::unique_ptr<Processor> create(InviwoApplication* app) override {
         auto proc = createProcessor(app);
-        auto p = std::unique_ptr<Processor>(proc.cast<Processor*>());
-        proc.release();
+        auto p = proc.cast<std::unique_ptr<Processor>>();
```

Comment:
Should this not be shared_ptr<Processor> now?

Reply:
the factories still uses unique_ptrs